src/main/scala/jigg/pipeline/RegexDocumentAnnotator.scala

```diff
@@ -0,0 +1,29 @@
+package jigg.pipeline
+
+import java.util.Properties
+import scala.xml.Node
+
+class RegexDocumentAnnotator(override val name: String, override val props: Properties) extends Annotator {
+  private[this] var documentID: Int = 0
+
+  override def annotate(annotation: Node): Node = {
+
+    def newDocumentID(): String = {
+      val new_id = "d" + documentID
+      documentID += 1
+      new_id
+    }
+
+    val raw = annotation.text
+    val reg = """\n{2,}"""
```

Comment:
It is desirable that user can customize the regular expression as in `RegexSentenceAnnotator`.
The expression would be customized via `@Prop` variable.